ec2/auto_scaling_group/templates/scripts/asg_user_data.sh

```
@@ -0,0 +1,330 @@
+        log "ERROR: Failed to start CloudWatch Agent with configuration from Parameter Store"
+    fi
+else
+    log "WARNING: CloudWatch Agent configuration parameter '${cloudwatch_config_parameter_name}' not found in Parameter Store"
+fi
+%{ endif ~}
+
+# ==================================================
+# Mackerel Agent設定
+# ==================================================
+
+%{ if mackerel_api_key != "" ~}
```

Shellcheck (0.10.0)

[warning] This is interpreted as a command name ending with '{'. Double check syntax.

(SC2288)